blank_react_project/src/config.js

```diff
@@ -1,67 +1,65 @@
-(function() {
-    const CONTRACT_NAME = 'react-template'; /* TODO: fill this in!*/
-    const DEFAULT_ENV = 'development'; 
-    
-    function getConfig(env) {
-        switch (env) {
+const CONTRACT_NAME = process.env.CONTRACT_NAME || 'react-template'; /* TODO: fill this in!*/
 
-        case 'production':
-        case 'development':
-            return {
-                networkId: 'default',
-                nodeUrl: 'https://rpc.nearprotocol.com',
-                contractName: CONTRACT_NAME,
-                walletUrl: 'https://wallet.nearprotocol.com',
-            };
-        case 'staging':
-            return {
-                networkId: 'staging',
-                nodeUrl: 'https://staging-rpc.nearprotocol.com/',
-                contractName: CONTRACT_NAME,
-                walletUrl: 'https://near-wallet-staging.onrender.com',
-            };
-        case 'local':
-            return {
-                networkId: 'local',
-                nodeUrl: 'http://localhost:3030',
-                keyPath: `${process.env.HOME}/.near/validator_key.json`,
-                walletUrl: 'http://localhost:4000/wallet',
-                contractName: CONTRACT_NAME,
-            };
-        case 'test':
-            return {
-                networkId: 'local',
-                nodeUrl: 'http://localhost:3030',
-                contractName: CONTRACT_NAME,
-                masterAccount: 'test.near',
-            };
-        case 'test-remote':
-        case 'ci':
-            return {
-                networkId: 'shared-test',
-                nodeUrl: 'http://shared-test.nearprotocol.com:3030',
-                contractName: CONTRACT_NAME,
-                masterAccount: 'test.near',
-            };
-        case 'ci-staging':
-            return {
-                networkId: 'shared-test-staging',
-                nodeUrl: 'http://staging-shared-test.nearprotocol.com:3030',
-                contractName: CONTRACT_NAME,
-                masterAccount: 'test.near',
-            };
-        default:
-            throw Error(`Unconfigured environment '${env}'. Can be configured in src/config.js.`);
-        }
-    }
-    
-    let Cookies = require('js-cookie');
```

Comment:
excellent. removing this `require` will fix the issue we discussed before, right @vgrichina? (The one where we used repl to see the true error of why near-shell was acting strange.

Reply:
yes